tests/e2e/TestConstants.ts

```diff
@@ -192,5 +197,10 @@ export const TestConstants = {
     /**
      * Log level settings, possible variants: 'INFO' (by default), 'DEBUG', 'TRACE'.
      */
-    TS_SELENIUM_LOG_LEVEL: process.env.TS_SELENIUM_LOG_LEVEL || 'INFO'
+    TS_SELENIUM_LOG_LEVEL: process.env.TS_SELENIUM_LOG_LEVEL || 'INFO',
+
+    /**
+     * Running test suite
+     */
+    TEST_SUITE: process.env.TEST_SUITE || ''
```

Comment:
Please align with
```
Happy Path test suite will be run by default when executing a docker run command. If you want to run another test suite, you can specify that via variable TEST_SUITE. Available tests are e.g. test-happy-path, test-che-install and test-wkspc-creation-and-ls.
```
from
https://github.com/eclipse/che/blob/master/dockerfiles/e2e/README.md
 - set default value to "test-happy-path"
 - add possible variants to the comments